packaging/datadog-agent/source/setup_agent.sh

```diff
@@ -389,18 +403,21 @@ tar -xz -C "$DD_HOME/agent" --strip-components 1 -f "$DD_HOME/agent.tar.gz"
 rm -f "$DD_HOME/agent.tar.gz"
 print_done
 
+# get the version from the actual config file in the branch
+AGENT_VERSION=$(cat $DD_HOME/agent/config.py | grep AGENT_VERSION | cut -f2 -d'=')
```

Comment:
`grep AGENT_VERSION` returns 2 lines here, we should probably use `grep "AGENT_VERSION ="` instead (it's a bit brittle, if for some reason we use `AGENT_VERSION=` in `config.py` in the future then the grep would fail, so it might be worth adding a comment in `config.py`...)

We could probably do the same thing for `JMX_VERSION` to be on the safe side.

Another solution: use `python`  to import `config.py` and print out these 2 versions. Probably safer.